app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/CurrentWorkoutActivity.java

```diff
@@ -134,6 +138,16 @@ public void onClick(View view) {
             }
         });
 
+        saveButton.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View view) {
+                if (!isPaused) {
+                    pauseButton.performClick();
```

Comment:
I think it would be cleaner if there was a dedicated pause() function that handles the check for `isPaused`
I can add this functionality when working on the save button.